src/impl.cpp

```diff
@@ -513,6 +513,10 @@ Manifold::Impl Manifold::Impl::Transform(const mat3x4& transform_) const {
     result.status_ = status_;
     return result;
   }
+  if (!all(isfinite(transform_))) {
+    result.MarkFailure(Error::NonFiniteVertex);
```

Comment:
Added this to catch any kind of bad transform.

Reply:
Apparently we need to make it explicit, i.e. `all(la::isfinite(transform_))` to make msvc happy.

Reply:
fwiw, in BRL-CAD when we use C++ we generally are explicit with the namespaces to avoid lookup surprises.  It's definitely a readability tradeoff though.